cpp/Untitled1.cpp

```diff
@@ -0,0 +1,15 @@
+// it makes difference while using 9 or 9.0 due to type casting.
```

Comment:
For eg change the name of this file from untitled1.cpp to something that defines functionality of the code

Reply:
Temperature